transform.js

```diff
@@ -10,6 +10,7 @@ const path = require('path')
 const fs = require('fs')
 
 const sheetify = require('./index')
+const SUPPORTED_VIEWS = require('./lib/supported-views')
```

Comment:
maybe views is not the best name for this, i thought in nanohtml it's referring to view libraries

Reply:
I agree, this isn't the best name for it.  Let me think of a better name.

Reply:
Ok I renamed this to `SUPPORTED_SYNONYMS`.